src/components/ec-radio-btn/ec-radio-btn.vue

```diff
@@ -0,0 +1,349 @@
+<template>
```

Comment:
We need to rework this component as it is not what we agreed on in the refinement. It should represent a single radio button using a single prop `name` to put multiple radio buttons into a group instead of passing multiple `options` as a prop. the v-model should handle the value updates.

It should work exactly like the ec-checkbox:
1. Have a native input type of radio in the HTML that is visually hidden from the user but all interactions are happening through that component.
2. Display a fake icon that represents the state of radio button when is selected/deselected/focused/disabled. Click the real input when the fake icon is clicked and the browser should handle all updates.

Follow the guide from the Vue docs: https://vuejs.org/guide/essentials/forms.html#radio

```
<ec-radio-btn v-model="value" name="my-button" label="One" />
<ec-radio-btn v-model="value" name="my-button" label="Two" />
```

Reply:
I see, this is the point of my RFC I'm going to create about design system and how components should be done in figma and storybook. TLDR: they should be exactly the same, name, props etc. We could add something, but props described in Figma should be the same in our Storybook, therefore designers, could do design review(including component behavior) and there will be a contract between these parts of developing process. This is why I think we could rename this component to EcRadioButtonGroup(in Figma too) and use it.

Reply:
turned to be to a single radio-button. later it will be created a radio-button-group component where the radio-button will be re-used